### ISSUE TITLE
[JENKINS-67962] The giant "No builds" message is shown even with a pending build

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/jenkins#17911]
<p>This looks unintentional.</p>

---
<details><summary><i>Originally reported by <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck">danielbeck</a>, imported from: <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-67962" target="_blank">The giant "No builds" message is shown even with a pending build</a></i></summary>
<i><ul>
<li><b>assignee</b>: <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=felix_416">felix_416</a>
<li><b>status</b>: Closed
<li><b>priority</b>: Minor
<li><b>component(s)</b>: core
<li><b>label(s)</b>: regression, ux
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2022-04-12T20:09:52+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 1
<li><b>imported</b>: 2025-11-24
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>This looks unintentional.</p></pre>
</details>
</details>
<ul><li><i>environment</i>: <code>2.337</code></li></ul>
<details><summary><i>3 attachments</i></summary>

- [Screenshot 2022-03-05 at 10.16.08.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/57393/Screenshot%202022-03-05%20at%2010.16.08.png)
  > ![Screenshot 2022-03-05 at 10.16.08.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/57393/Screenshot%202022-03-05%20at%2010.16.08.png)
- [ajax.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/57395/ajax.png)
  > ![ajax.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/57395/ajax.png)
- [initial.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/57396/initial.png)
  > ![initial.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/57396/initial.png)
</details>

**Comment by jenkins-infra-bot** [comment]
- _Remote link associated with this issue:_


  - [https://github.com/jenkinsci/jenkins/pull/6461](https://github.com/jenkinsci/jenkins/pull/6461)

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-67962?focusedId=422682&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-422682">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>The behavior is inconsistent between an initial page load (showing both the message and the queued build) and an AJAX reloaded widget (showing only the queued build).</p>

<p>So even if the reported behavior is for some reason not considered a bug, the big placeholder disappearing through AJAX update would then be the bug.</p>
</pre>
</details>

</ul>
</details>

<p>The behavior is inconsistent between an initial page load (showing both the message and the queued build) and an AJAX reloaded widget (showing only the queued build).</p>

<p>So even if the reported behavior is for some reason not considered a bug, the big placeholder disappearing through AJAX update would then be the bug.</p>
